lib/tasks/sync_governance.rake

```diff
@@ -0,0 +1,26 @@
+require "task_lock"
+
+task sync_governance: :environment do
+  $stdout.sync = true
+  TaskLock.with_lock!(:sync_governance) do
+    puts "Setting Tezos Chain"
+    chain = Tezos::Chain.first
```

Comment:
Should it not go through all Tezos::Chain's? If there's always only 1 (which would be fine) then probably don't need the table at all eh?

Reply:
fixed to handle multiple chains.